packaging/bin/pki-enroll-pkcs12.sh

```diff
@@ -59,7 +59,7 @@ enroll() {
 		|| die "Cannot sign request"
 
 	touch "${pkcs12}"
-	chmod go-rwx "${pkcs12}" || die "Cannot set PKCS#12 permissions"
+	chmod 0644 "${pkcs12}" || die "Cannot set PKCS#12 permissions"
```

Comment:
Shouldn't it be 0600 based on above line?

Reply:
fixed in current commit